src/internal/connector/data_collections_test.go

```diff
@@ -334,7 +335,7 @@ func (suite *ConnectorCreateSharePointCollectionIntegrationSuite) TestCreateShar
 	)
 
 	sel := selectors.NewSharePointBackup(siteIDs)
-	sel.Include(sel.LibraryFolders([]string{"foo"}, selectors.PrefixMatch()))
+	sel.Include(sel.LibraryFolders([]string{"zzazil"}, selectors.PrefixMatch()))
```

Comment:
What does this magic keyword do?

Reply:
That magic keyword reminds me that I need to come back and investigate a change in that location.  Usually.  Eventually.